dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -272,12 +275,6 @@ def _add_helper(
         :param other2: profile2 being added to self
         :return: None
         """
-        if not (
-            isinstance(other1, NumericStatsMixin)
-            and isinstance(other2, NumericStatsMixin)
-        ):
-            raise ValueError("Parameters must be of type NumericStatsMixin.")
```

Comment:
I think we still need this for the error, no? Or was this added for the LSP last time and this is handled by the classes using the mixins?